zc_install/index.php

```diff
@@ -9,7 +9,7 @@
   // Actual version check is more strict; this is just to start the program
   // For true minimum, see includes/systemChecks.yml under checkPhpVersionMin
   if (PHP_VERSION_ID < 70100) { // only checking for 7.1 here as a basic check. zc_install itself doesn't need higher than this at this time
```

Comment:
Note: This check in `zc_install/index.php` is merely for zc_install to "run".  We don't require super-modern PHP versions here in zc_install, which is good, so that the usual zc_install experience can still operate friendly until the PHP version inspection is run, at which point that and any other errors are displayed.